src/controllers/organizations.js

```diff
@@ -0,0 +1,176 @@
+const { default: mongoose } = require('mongoose');
+const { Organization, User } = require('../models/user');
+const Fund = require('../models/fund');
```

Comment:
My eyes hurt! I went over the code multi times to know where you used fund schema haha! But I found nothing!